pcdsdevices/lasers/btps.py

```diff
@@ -1,8 +1,36 @@
+from __future__ import annotations
+
+from typing import Dict, List, Optional, Tuple, cast
+
 from ophyd.device import Component as Cpt
 from ophyd.device import Device
+from ophyd.signal import EpicsSignalRO
+from ophyd.status import AndStatus, MoveStatus
 
+from pcdsdevices.valve import VGC
+
+from ..device import UnrelatedComponent as UCpt
+from ..epics_motor import SmarAct
 from ..interface import BaseInterface
 from ..signal import PytmcSignal
+from . import btms_config as btms
+from .btms_config import (BtmsSourceState, BtmsState, DestinationPosition,
+                          MoveError, SourcePosition)
+
+
+class BtpsVGC(VGC):
+    """
+    VGC subclass with 'valve_position' component added.
+    """
+    # TODO: this may be pushed into ValveBase, but need to check with others
+    # first
```

Comment:
REF: https://github.com/pcdshub/pcdsdevices/pull/1044
We could absolutely merge 1044 I think